src/PrestaShopBundle/Controller/Admin/Sell/Order/CreditSlipController.php

```diff
@@ -84,7 +84,6 @@ public function indexAction(
         $pdfByDateForm = $this->createForm(GeneratePdfByDateType::class, [], [
             'method' => Request::METHOD_GET,
         ]);
-        $pdfByDateForm->handleRequest($request);
 
```

Comment:
Removing because it tried to handle generatePpdfByDate form in index page, when it should only do in it's own action.